server/src/test/java/org/elasticsearch/index/replication/IndexLevelReplicationTests.java

```diff
@@ -317,7 +317,7 @@ public void testRequestFailureReplication() throws Exception {
     public void testSeqNoCollision() throws Exception {
         try (ReplicationGroup shards = createGroup(2)) {
             shards.startAll();
-            int initDocs = shards.indexDocs(randomInt(10));
+            int initDocs = shards.indexDocs(randomIntBetween(2, 10));
```

Comment:
why move from 0?

Reply:
`initDocs = 0` reproduces the case while primary has no any docs indexed while replica has some indexed docs. definitely have to keep it from 0